package.json

```diff
@@ -68,5 +68,8 @@
     "webpack": "^4.29.3",
     "webpack-cli": "^3.2.3",
     "webpack-merge": "^4.2.1"
-  }
+  },
+  "jest": {
+       "testURL": "http://localhost:5050/"
```

Comment:
is there any specific reason for this 🤔

Reply:
This is necessary to get the path during tests. It serves as the base path. It was used to run the test for the NotFound component

Reply:
okay 👍🏿